src/labels/CustomLabels.labels

```diff
@@ -13101,4 +13173,20 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>Resource Partners link label in Get Started Resources section</shortDescription>
         <value>Find a Partner</value>
     </labels>
+    <labels>
+        <fullName>accordionSection_ToggleInstructionsWhenOpen</fullName>
```

Comment:
These need to be place in the labels alphabetically, so these should be up towards the top with the label names that start with "a"